tests/system/test_database_api.py

```diff
@@ -159,6 +161,48 @@ def test_create_database_with_default_leader_success(
             assert result[0] == default_leader
 
 
+def test_iam_policy(not_emulator, shared_instance, databases_to_delete):
+    pool = spanner_v1.BurstyPool(labels={"testcase": "iam_policy"})
+    temp_db_id = _helpers.unique_id("iam_db", separator="_")
+    create_table = (
+        f"CREATE TABLE policy (\n"
+        f"    Id      STRING(36) NOT NULL,\n"
+        f"    Field1  STRING(36) NOT NULL\n"
+        f") PRIMARY KEY (Id)"
+    )
+    create_role = f"CREATE ROLE parent"
+
+    temp_db = shared_instance.database(
+        temp_db_id,
+        ddl_statements=[create_table, create_role],
+        pool=pool,
+    )
+    create_op = temp_db.create()
+    databases_to_delete.append(temp_db)
+    create_op.result(DBAPI_OPERATION_TIMEOUT)
+    policy = temp_db.get_iam_policy(3)
+
+    assert policy.version == 0
+    assert policy.etag == b"\x00 \x01"
+
+    new_binding = policy_pb2.Binding(
+        role="roles/spanner.fineGrainedAccessUser",
+        members=["user:asthamohta@google.com"],
```

Comment:
Should we explicit mention human users ("user:asthamohta@google.com")? Maybe we should use a non-human user?

Reply:
Yes we plan on changing this for GA to a value that user can set up